view/frontend/templates/form/multishipping/abstract-form.phtml

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+?>
+<!-- ko template: getTemplate() --><!-- /ko -->
+<script>
+    require([
+        'uiLayout',
+        'jquery',
+        'Adyen_Payment/js/model/adyen-payment-service'
+    ], function (layout, $, adyenPaymentService) {
+        $(function () {
+
+            let allowedPaymentMethods = [
+                // 'amazonpay',
+                // 'applepay',
+                // 'bcmc_mobile',
+                // 'boleto',
+                // 'dotpay',
+                // 'facilypay_3x',
+                // 'googlepay',
+                'ideal',
+                'klarna',
+                // 'multibanco',
+                // 'paypal'
+            ];
+
+            // TODO after ECP-8516
+            // let paymentMethodsObserver = adyenPaymentService.getPaymentMethods();
+            // paymentMethodsObserver.subscribe(
+            //     function(paymentMethodsResponse) {
+            //         let filteredPaymentMethods = paymentMethodsResponse.paymentMethodsResponse.paymentMethods.filter(paymentMethod => {
+            //             return allowedPaymentMethods.includes(paymentMethod.type);
+            //         });
```

Comment:
Could you please remove this TODO block. This will be handled in a separate ticket. Let's keep this list hardcoded on line 22 for a while.